scipy/optimize/_differentialevolution.py

```diff
@@ -488,6 +490,8 @@ class DifferentialEvolutionSolver:
         method is used to polish the best population member at the end, which
         can improve the minimization slightly. If a constrained problem is
         being studied then the `trust-constr` method is used instead.
+        Please note that for large problems with many constraints, polishing
```

Comment:
As a British person I like the level of politeness, but I think we should change this to:

> For large problems ...